### ISSUE TITLE
Instance - LBRS - Highlord Omokk

### ISSUE THREAD

**Issue by Bilbo79** [scotty0100/wowascension-issues#480]
Name: Ocholoco
Account Name: Bilbo79
Map & Area:   LBRS, just past the entrance door to Highlord Omokk's room
How to replicate the bug:   Pull Omokk back to this location to fight
What happens:

a)  Omokk can be 1 shot killed.  We have seen it on 3 of the last 4 kills.  Here is how I think it must be reproduced.
- Fight the ogres in the large entrance area, just beyond the last doorway, leading to the Highlords room.
- When pulling back, target and pull Omokk.  If you pull either of his two adds (next to him), then the fight goes normally.
- After pulling, run back to the entrance, and LOS the group by going behind one of the wall pillars.  The initial idea was to get the caster to come all the way out.

But what happens is that both the caster and the melee add disengage, and run back to their initial position.

And that leaves only Omokk.  Engage Omokk, and have dps start in on him.  He either dies instantly , or when the first Frostbolt comes in.

I really need to take a SS of the combat log.  I will add that when I do this next.

What should happen:

a)  He should not be killed with 1 shot.

Located on WoWHead Map (if applicable):   LBRS, in Highlord Omokk room
Source (proof, if applicable): Personal Experience

Extra Information:

**Comment by Bilbo79** [comment]
2 more tries, and I cannot reproduce this.  I am not exactly sure now.  I just know that it has happened at least 3 or 4 times recently.  We have killed this 10 times leading up to this and not seen it once.  Recently seeing it a good deal.  And the last 2 not seeing it.  I am not sure how to make it happen.

**Comment by Tobschinski** [comment]
There is a chance that Highlord Omokk is either a 58 or a level 59 boss. (this is incorrect, he should have a fix level)
Therefore, he has a different amount of health depending on his level.

His minimum HP, which is chosen if he rolls level 58 is 22,272 health, the maximum 2,227 - which leads to the described issue.

**Comment by Bilbo79** [comment]
Yeah, I bet you are on to something.  2,227 sounds about right.  The fight is literally over before it starts, since 3 mages open fire and hit for (750-1500) each, or 2250-4500 total.

I will get a SS.  14 kills, and no Skyshroud leggings for any of my 3 mages.  Yeah, we will be back.  Lots.  And hopefully I will have better information how to reproduce this consistently.

**Comment by Bilbo79** [comment]
Yes, you have the answer.  This is his health if he spawns as a 58.  SS:

http://i167.photobucket.com/albums/u132/Bilbo79/Omokk_HP_58_zps173ceb8d.jpg
